site/app/templates/polls/PollForm.twig

```diff
@@ -202,6 +221,36 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
+    function toggleTimerInputs() {
+        var enableTimerCheckbox = document.getElementById("enable-timer");
+        var timerInputs = document.getElementById("timer-inputs").querySelectorAll(".poll-timer");
```

Comment:
Using `const` or `let` (as appropriate) is better than `var` in JS.

```suggestion
        const enableTimerCheckbox = document.getElementById("enable-timer");
        const timerInputs = document.getElementById("timer-inputs").querySelectorAll(".poll-timer");
```

Reply:
Changed this